google-cloud-tools-plugin/resources/config.properties

```diff
@@ -19,3 +19,5 @@ feature.appengine.flex=true
 
 # Should be used in any Notifications explicitly produced by the plugin.
 notifications.plugin.groupdisplayid=Google App Engine plugin
+
+cloudsdk.min.version=131.0.0
```

Comment:
No issue, if you feel strongly I'm happy to add the newline. (Though I'll point out that it's not mentioned in the style guide).